prisondersdilemma.py

```diff
@@ -11,182 +11,260 @@
-		win.flip()
+def defect:
+	'''opponent always defects'''
+	opponent.draw()
+	participant.draw()
+	deftxt.draw()
+	win.flip()
+	wait(4) #seconds
+	connecttxt.draw
+	#sentence draw 
+	win.flip()
 	
-	#random coop or defect
-	elif strategy == 3:
-		shuffle(randostrategy)
+def random:
+	'''opponent cooperates or defects on a random basis'''
+	shuffle(randostrategy)
 		if randostrategy == 1:
 			opponent.draw()
+			participant.draw()
 			cooptxt.draw()
 			win.flip() 
 			wait(4)#seconds
 			#sentence draw 
 			win.flip()
-		elif: 
+		else: 
 			opponent.draw()
+			participant.draw()
 			deftxt.draw()
 			win.flip()
 			wait(4) #seconds
 			connecttxt.draw
 			#sentence draw 
 			win.flip()
-	
-	elif strategy == 4:
-		if condlist[i] == 1:
+			
+def nicetittat:
+	'''nice tit for that: opponent cooperates in first round, then copies participants moves'''
+	if condlist[i] == 1:
 			opponent.draw()
+			participant.draw()
 			cooptxt.draw()
 			win.flip() 
 			wait(4)#seconds
 			connecttxt.draw
 			#sentence draw 
 			win.flip()
-		elif condlist[i] > 1:
+		else condlist[i] > 1:
 			if resp == 'right':
 				opponent.draw()
+				participant.draw()
 				cooptxt.draw()
 				win.flip() 
 				wait(4)#seconds
 				connecttxt.draw
 				#sentence draw 
 				win.flip()
-			elif resp == 'left':
+			else resp == 'left':
 				opponent.draw()
+				participant.draw()
 				deftxt.draw()
 				win.flip() 
 				wait(4)#seconds
 				connecttxt.draw
 				#sentence draw 
 				win.flip()
-	elif strategy == 5:
-		if condlist[i] == 1:
+	
+def susptittat:
+	'''suspicious tit for that: opponent defects in the first round, then copies the participant's moves'''
+	if condlist[i] == 1:
 			opponent.draw()
+			participant.draw()
 			deftxt.draw()
 			win.flip() 
 			wait(4)#seconds
 			connecttxt.draw
 			#sentence draw 
 			win.flip()
-		elif condlist[i] > 1:
+		else condlist[i] > 1:
 			if resp == 'right':
 				opponent.draw()
+				participant.draw()
 				cooptxt.draw()
 				win.flip() 
 				wait(4)#seconds
 				connecttxt.draw
 				#sentence draw 
 				win.flip()
-			elif resp == 'left':
+			else resp == 'left':
 				opponent.draw()
+				participant.draw()
 				deftxt.draw()
 				win.flip() 
 				wait(4)#seconds
 				connecttxt.draw
 				#sentence draw 
 				win.flip()
+	
+
+#starting the game
+background_image.draw #background prisonwall
+intro_txt.draw()
+win.flip()
+waitKeys(keyList = ('space'))
+
+#choosing avatar that represents participant 
+avat_txt.draw()
+partOverview.draw()
+win.flip()
+response = waitKeys(keyList = ('a', 'b','c','d'))
+
+shuffle(condlist) #randomly assigning condition (in group, out group, anonymous opponent)
+
+if condlist == 1 & response ==[a]:#participant is female caucasian and plays with in-group member
+	opponent.setImage("opponent2.png")
+	participant.setImage("partA") 
+	#tipps and example on how to do this for every condition on possible choice of participant avatar in a function, my 2 weeks of python experience didn't get me too far on this
```

Comment:
The only thing I can come up with is the group the responses that lead to the same opponent together so that you have two write if statements for two groups of responses instead of four different responses individually: 

responses = ('a', 'b', 'c', 'd')  
ac = ('a', 'c') # group 1 
bd = ('b', 'd') # group 2 
index = [ac, bd] # the possible responses

shuffle(condlist) 
con = condlist[0]
res = # this is their avatar, whatever it is
if con == 1 and response in index[0]: # if the response is in the first response group